Formula/flann.rb

```diff
@@ -1,9 +1,9 @@
 class Flann < Formula
   desc "Fast Library for Approximate Nearest Neighbors"
   homepage "https://www.cs.ubc.ca/research/flann/"
-  url "https://github.com/mariusmuja/flann/archive/1.9.1.tar.gz"
-  sha256 "b23b5f4e71139faa3bcb39e6bbcc76967fbaf308c4ee9d4f5bfbeceaa76cc5d3"
-  revision 9
+  url "https://github.com/xlingsky/flann/archive/m1.9.1.tar.gz"
```

Comment:
Why is the source changing?

Reply:
Please see  #[https://github.com/mariusmuja/flann/issues/408](url)
There is no new release from the original author.
So I forked the repo and released the new one to support C++17.

Reply:
That's not good enough, unless you have a redirect from the original source we don't change the sources of formula.